monkey/monkey_island/cc/models/monkey.py

```diff
@@ -68,6 +68,19 @@ def get_os(self):
             os = "windows"
         return os
 
+    @staticmethod
+    def get_tunneled_monkeys():
```

Comment:
Add unit tests? It fun and easy using mongoengine 🚒

Reply:
Ok, I'll try that.